common/source/docs/common-cubepilot-cubenodeeth.rst

```diff
@@ -0,0 +1,102 @@
+    :target: ../_images/cubepilot-cubenodeeth-pinout.png
+    :height: 450px
+
+The general rule when connecting ethernet devices is:
+
+- When connecting a device to a switch the pins should be connected straight through (e.g. RX+ to RX+, RX- to RX-, TX+ to TX+, TX- to TX-)
+- When connecting a device to a device the pins should be crossed (e.g. RX+ to TX+, RX- to TX-, TX+ to RX+, TX- to RX-)
+
+Install ArduPilot with PPP Support
+==================================
+
+`PPP <https://en.wikipedia.org/wiki/Point-to-Point_Protocol_over_Ethernet>`__ allows an autopilot to communicate over Ethernet using a serial port but with the exception of the CubePilot CubeOrangePlus, PPP is not included by default in the ArduPilot firmware.
+
+- Use the `Custom Firmware Build Server <https://custom.ardupilot.org/>`__ to build a firmware that includes PPP support
+
+.. image:: ../../../images/build-server-ppp.jpg
+    :target: ../_images/build-server-ppp.jpg
+
+- Alternatively :ref:`build the code yourself <building-the-code>` and enable PPP by adding ``--enable-PPP`` to the waf configuration step
+- :ref:`Install the firmware on the autopilot <common-loading-firmware-onto-pixhawk>`
+
+Autopilot Configuration
+=======================
+
+See the ``PPP configuration`` and ``ArduPilot Port Configuration`` sections of :ref:`common-network` but in short set these parameters:
+
+- :ref:`CAN_P1_DRIVER <CAN_P1_DRIVER>` = 1 (First driver)
+- :ref:`CAN_D1_DRIVER <CAN_D1_PROTOCOL>` = 1 (DroneCAN)
+- :ref:`NET_ENABLE <NET_ENABLE>` = 1
+- :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` = 48 (PPP)
+- :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` = 12500000 (12.5MBaud)
+
+To allow the autopilot to eventually communicate with other ethernet devices, the :ref:`NET_P1_TYPE <NET_P1_TYPE>`, :ref:`NET_P1_PROTOCOL <NET_P1_PROTOCOL>`, etc parameters will need to be set.
+More details on the settings required can be found in the "Ethernet Conectivity" sections of the ArduPilot wiki page for the device.  For example :ref:`Siyi camera setups can be found on this page <common-siyi-zr10-gimbal>`
+
+CubeNodeETH Setup
+=================
+
+.. image:: ../../../images/cubepilot-cubenodeeth-params.png
+    :target: ../_images/cubepilot-cubenodeeth-params.png
+
+The CubeNodeETH's parameters can be set using Mission Planner's DroneCAN screen or the :ref:`DroneCAN GUI tool <common-uavcan-gui>`.  The following instructions assume Mission Planner is used:
+
+- Connect to the AutoPilot with Mission Planner 
+- Open the Setup, Optional Hardware, DroneCAN/UAVCAN screen
+- From the top-left drop-down select MAVLinkCAN1 (or MAVLinkCAN2 if the autopilot's CAN2 is used)
+- Press "Connect" and the table should be populated with the available DroneCAN devices including "org.ardupilot.CubePilot-PPPGW"
+- Press the "Menu" button on the right side and select "Parameters"
+- Set the following parameters and press the "Write" button
+
+  - NET_DHCP = 0 (to disable dynamic IP assignment)
```

Comment:
hotlink all these parameters, please. Even though they are periph, they are still listed in the wiki param pages and linking flags name changes, even though that is unlikely

Reply:
yup, will do, thanks!  a bonanza of merged PRs just now, thanks!

Reply:
Txs for the reivew, I've updated the links on this new page and also the dronenet page